e2e/consul/input/consul_wi.nomad.hcl

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+job "example" {
+
```

Comment:
Right now the Windows client is disabled, but we'd like to re-enabled that in the future. This job probably won't run on Windows (because it's a Linux image), so let's add the `constraint` for Linux that we have on most of the other E2E jobs here too.

Reply:
oof good tip, I always forget about windows 😬

Reply:
addressed in [b66add3](https://github.com/hashicorp/nomad/pull/19500/commits/b66add3774db497cccde45410832b47ccf29d36c)